it/connectors/v3/getFinancialDetails/GetGetFinancialDetailsConnectorISpec.scala

```diff
@@ -23,10 +23,10 @@ import play.api.test.Helpers.{await, defaultAwaitTimeout}
 import uk.gov.hmrc.http.HeaderCarrier
 import utils.{ETMPWiremock, IntegrationSpecCommonBase}
 
-class GetFinancialDetailsConnectorISpec extends IntegrationSpecCommonBase with ETMPWiremock with FeatureSwitching {
+class GetGetFinancialDetailsConnectorISpec extends IntegrationSpecCommonBase with ETMPWiremock with FeatureSwitching {
```

Comment:
Could you change the class name and file name back to `GetFinancialDetailsConnectorISpec`